owsi-core/owsi-core-components/owsi-core-component-wicket-more/src/main/java/fr/openwide/core/wicket/more/rendering/Renderer.java

```diff
@@ -652,5 +654,116 @@ public String render(final T value, Locale locale) {
 			return Localizer.get().getString(resourceKeyBuilder.toString(), null, mapModel, locale, null, null);
 		}
 	}
-
+	
+	/**
+	 * <p>A simplified version of the {@link #range(String, Renderer, Renderer)} function.
+	 * <p>Returns a static renderer for {@link Range} elements in which values have no units.
```

Comment:
Maybe a better way of putting it would be the following:

```
Returns a renderer for {@link Range} elements in which there is no notion of "unit-less" bound rendering.
<p>Use this if you bounds have no unit whatsoever ("From 2016/03/02 to 2016/03/03") or if there's no need to prevent unit repetition ("From 2km to 3km")
<p>If you **do** need to prevent unit repetition, for instance in "From 2 persons to 3 persons", which is better written "From 2 to 3 persons", use the full {@link #range(String, Renderer, Renderer)} function.
```